docusaurus/docs/welcome/instructions/app-icon.md

```diff
@@ -0,0 +1,164 @@
+
+> ℹ️ Although [Android Asset Studio](https://github.com/romannurik/AndroidAssetStudio) was also considered, we decided to adopt [Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio) because it's the official tool from Google, it's built in into Android Studio and it not only generates the assets but it also places them in the correct folders.
+
+#### iOS
+
+The iOS assets can be generated using [App Icon Generator](https://appicon.co/).
+On the "iOS and macOS" section, only `iPhone` and `iPad` should be selected.
+It is possible to either click and browse the image source or simply drag and drop it.
+After selecting the image, click on the "Generate" button to download the assets.
+Note that the filenames of the generated assets should be renamed according to the following pattern: `<resolution>@x<multiplier>`
+
+> ℹ️  Unfortunately there isn't an official tool for app icon generation on iOS.
+Even though there are Design related tools that solves this problem, there aren't similar ones which are non-designers friendly and significantly better than the recommended tool.
+
+### Output assets location
+
+#### Android
+
+> 💡 Note that if you used [Image Asset Studio](https://developer.android.com/studio/write/image-asset-studio), you won't have to bother about this, because the tool automatically places the assets into the correct locations.
+
+##### Square icons:
+
+-   **mdpi (48x48@1x):** `android/app/src/main/res/mipmap-mdpi/app_icon.png`
+-   **hdpi (72x72@1\.5x):** `android/app/src/main/res/mipmap-hdpi/app_icon.png`
+-   **xhdpi (96x96@2x):** `android/app/src/main/res/mipmap-xhdpi/app_icon.png`
+-   **xxhdpi (144x144@3x):** `android/app/src/main/res/mipmap-xxhdpi/app_icon.png`
+-   **xxxhdpi (192x192@4x):** `android/app/src/main/res/mipmap-xxxhdpi/app_icon.png`
+
+##### Round icons:
+
+-   **mdpi (48x48@1x):** `android/app/src/main/res/mipmap-mdpi/app_icon_round.png`
+-   **hdpi (72x72@1\.5x):** `android/app/src/main/res/mipmap-hdpi/app_icon_round.png`
+-   **xhdpi (96x96@2x):** `android/app/src/main/res/mipmap-xhdpi/app_icon_round.png`
+-   **xxhdpi (144x144@3x):** `android/app/src/main/res/mipmap-xxhdpi/app_icon_round.png`
+-   **xxxhdpi (192x192@4x):** `android/app/src/main/res/mipmap-xxxhdpi/app_icon_round.png`
+
+After moving the assets to the following locations, there is nothing more to do.
+Both "Square" and "Circle" shapes should work.
+
+#### iOS
+
+There are two options: either you manually move the icons and reflect the changes on the `ios/reactNativeWithMoxy/Images.xcassets/AppIcon.appiconset/Contents.json` file or
+drag and drop the assets using XCode, which will automatically update it.
+
+Using XCode:
+
+On the project tree, navigate to `reactNativeWithMoxy > Images.xcassets`.
+
+![XCode AppIcon](/img/xcode-app-icon.png)
+
+Now you should be able to drag and drop each asset to its correct location.
+
+![XCode AppIcon Drag and Drop](/img/xcode-app-icon-drag-n-drop.png)
```

Comment:
I should add a screenshot to demonstrate how the Xcode looks after the icons are in place